tools/releaseBuild/release.yml

```
@@ -0,0 +1,173 @@
+name: PlatyPS-Release-$(Build.BuildId)
+trigger: none
+
+pr:
+  branches:
+    include:
+    - v2
+    - release/v2/*
+
+variables:
+  - group: ESRP
+  - name: DOTNET_CLI_TELEMETRY_OPTOUT
+    value: 1
+  - name: POWERSHELL_TELEMETRY_OPTOUT
+    value: 1
+
+resources:
+  repositories:
+  - repository: ComplianceRepo
+    type: github
+    endpoint: ComplianceGHRepo
+    name: PowerShell/compliance
+
+stages:
+- stage: Build
+  displayName: Build
+  pool:
+    vmImage: windows-latest
+  jobs:
+  - job: Build_Job
+    displayName: Build PlatyPS
+    steps:
+    - checkout: self
+      clean: true
+    - pwsh: |
+        $versionString = if ($env:RELEASE_VERSION -eq 'fromBranch') {
+          $branch = $env:BUILD_SOURCEBRANCH
+          $branchOnly = $branch -replace '^refs/heads/'
+          $branchOnly -replace '^.*(release/v2[-/])'
+        }
+        else {
+            $env:RELEASE_VERSION
+        }
+
+        $nugetVersion = if ($versionString.StartsWith('v')) {
+            $versionString.Substring(1)
+        }
+        else {
+            $versionString
+        }
+
+        $vstsCommandString = "vso[task.setvariable variable=Version]$nugetVersion"
+        Write-Verbose -Message "setting Version to $releaseTag" -Verbose
+        Write-Host -Object "##$vstsCommandString"
+      displayName: Set NuGet package version variable
+
+    - pwsh: |
+        ./build.ps1 -Clean -Configuration 'Release'
+      displayName: Execute build
+
+    - publish: '$(Build.SourcesDirectory)\out\platyPS'
+      artifact: build
+      displayName: Publish build package
+
+    - template: EsrpSign.yml@ComplianceRepo
+      parameters:
+        buildOutputPath: '$(Build.SourcesDirectory)\out\platyPS'
+        signOutputPath: '$(Pipeline.Workspace)\signed\codesign'
+        certificateId: 'CP-230012'
+        pattern: |
+          **\Microsoft.PowerShell.PlatyPS.dll
+          **\platyPS.psd1
+        useMinimatch: true
+
+    - template: EsrpSign.yml@ComplianceRepo
+      parameters:
+        buildOutputPath: '$(Build.SourcesDirectory)\out\platyPS'
+        signOutputPath: '$(Pipeline.Workspace)\signed\thirdparty'
+        certificateId: 'CP-231522'
```

there is a `displayName` parameter to name these, please use it
```suggestion
        certificateId: 'CP-231522'
        displayName: Third party signing
```

---

Added